cmd/neofs-cli/modules/bearer/create.go

```diff
@@ -52,33 +51,29 @@ func init() {
 
 	_ = cobra.MarkFlagRequired(createCmd.Flags(), issuedAtFlag)
 	_ = cobra.MarkFlagRequired(createCmd.Flags(), notValidBeforeFlag)
-	_ = cobra.MarkFlagRequired(createCmd.Flags(), expireAtFlag)
+	_ = cobra.MarkFlagRequired(createCmd.Flags(), commonflags.ExpireAt)
 	_ = cobra.MarkFlagRequired(createCmd.Flags(), ownerFlag)
 	_ = cobra.MarkFlagRequired(createCmd.Flags(), outFlag)
 }
 
-func createToken(cmd *cobra.Command, _ []string) error {
+func createToken(cmd *cobra.Command, _ []string) {
 	iat, iatRelative, err := common.ParseEpoch(cmd, issuedAtFlag)
-	if err != nil {
-		return err
-	}
-	exp, expRelative, err := common.ParseEpoch(cmd, expireAtFlag)
-	if err != nil {
-		return err
-	}
+	common.ExitOnErr(cmd, "can't parse --"+issuedAtFlag+" flag: %w", err)
+
+	exp, expRelative, err := common.ParseEpoch(cmd, commonflags.ExpireAt)
```

Comment:
does it still support a relative expiration on the `expire-at` flag?

Reply:
Yes, i think bearer command better stay the way it is, because it has 3 similar flags. Having 6 flags can be too hard for a user.